.github/workflows/pr.yml

```diff
@@ -0,0 +1,83 @@
+name: pr
```

Comment:
This is pretty much what `ci.yml` was previously, but now just for pull requests